File: API/API/Models/IMC.cs
using System;

namespace API.Models;

public class IMC
{
    public string? ImcId { get; set; }
    public float Peso { get; set; }
    public float Altura { get; set; }
    public Aluno? Aluno { get; set; } 
    public string? AlunoId { get; set; }
    public string? Classificacao { get; set; }
    public DateTime CriadoEm { get; set; } = DateTime.Now;
}

File: front/src/models/imc.ts
import { Aluno } from "./aluno"

export interface IMC{
    imcId? : string,
    peso : number,
    altura? : number,
    criadoEm? : string,
    classificacao : string,
    aluno? : Aluno, 
    alunoId : string
}
export default IMC;